packages/apptive_grid_core/lib/src/model/form/component/form_component.dart

```diff
@@ -74,17 +80,22 @@ class FormComponent<T extends DataEntity> {
         data == other.data &&
         options == other.options &&
         required == other.required &&
-        type == other.type;
+        type == other.type &&
+        enabled == other.enabled;
   }
 
   @override
   int get hashCode =>
-      Object.hash(field, property, data, options, required, type);
+      Object.hash(field, property, data, options, required, type, enabled);
 
   /// Mapping to a concrete implementation based on [json] and [schema]
   ///
   /// Throws an [ArgumentError] if not matching implementation is found.
-  static FormComponent fromJson(dynamic json, List<GridField> fields) {
+  static FormComponent fromJson(
+    dynamic json, {
+    required List<GridField> fields,
```

Comment:
I think we should keep this `fields` still as a positional argument to not introduce a breaking change.

Reply:
done